vendor/abeeway/abeeway-geolocation-module-pro-as923.yaml

```diff
@@ -1,13 +1,13 @@
 # LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
-macVersion: 1.0.2
+macVersion: 1.0.4
```

Comment:
Likewise, profiles cannot be changed.

If there are new hardware and firmware versions, please add the files.